elfi/visualization/visualization.py

```diff
@@ -397,3 +397,50 @@ def plot_discrepancy(gp, parameter_names, axes=None, **kwargs):
         axes[idx].set_axis_off()
 
     return axes
+
+
+def plot_gp(gp, parameter_names, axes=None, resol=50, const=None, **kwargs):
+    """Plot pairwise relationships as a matrix with parameters vs. discrepancy.
+
+    Parameters
+    ----------
+    gp : GPyRegression, required
+    parameter_names : dict, required
+        Parameter names from model.parameters dict('parameter_name':(lower, upper), ... )`
+    axes : matplotlib.axes.Axes, optional
```

Comment:
Single axis or an array? If single, how will it be used?

Reply:
`plt.Axes or arraylike of plt.Axes`. The usage is the same as in other functions in _visualization_ module.